eng/ci/templates/official/jobs/build-artifacts-windows.yml

```diff
@@ -2,6 +2,20 @@ jobs:
 - job: BuildArtifactsWindows
   displayName: Build Windows Artifacts
 
+  pool:
+    name: 1es-pool-azfunc
+    image: 1es-windows-2022 
+    os: windows
+
+  variables:
+    ${{ if or( eq( variables['Build.Reason'], 'PullRequest' ), and( not( contains( variables['Build.SourceBranch'], 'release/inproc6/4.' ) ), not( contains( variables['Build.SourceBranch'], 'release/inproc8/4.' ) ), not( contains( variables['Build.SourceBranch'], 'release/4.' ) ), not( contains( variables['Build.SourceBranch'], 'release/ExtensionsMetadataGenerator/' ) ) ) ) }}:
```

Comment:
Shall we remove the "release/inproc6/4" and "release/inproc8/4" usage here?

Reply:
I have updated that in a different PR: #10171